src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs

```diff
@@ -1,12 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
 //
 // Mono runtime native structs surfaced to managed code.
 //
 // Authors:
```

Comment:
@marek-safar - while we are updating headers, does it make sense to get rid of the authors of the file (has xamarin or ximian email ids) ?(This if for files we are updating here)

Reply:
We could if it makes the process easier

Reply:
I was wondering about that too, then I saw library also has files like this. For example, https://github.com/dotnet/runtime/blob/master/src/libraries/System.Drawing.Common/src/System/Drawing/Bitmap.Unix.cs

But if our ultimate goal is to get rid of those, I could do have my script doing that as well.

Reply:
Looking at a few coreclr files, I would suggest removing it for the files you are touching here

Reply:
we'll create a separate tracking issue to remove author names. Resolving this comment